app/src/layouts/calendar/index.ts

```diff
@@ -167,11 +167,11 @@ export default defineLayout<LayoutOptions>({
 					} else {
 						const primaryKey = info.event.id;
 
-						const endpoint = collection.value.startsWith('directus')
+						const route = collection.value.startsWith('directus_')
```

Comment:
this is a little unrelated drive-by change, but since the purpose is for navigating (router.push) to another page rather than an actual API endpoint, opted to rename it to `route` for clarity, as well as `directus` to `directus_` for consistency sake.